data/blog/2024-06-14-austrian-alps-trip.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Trip to Austrian alps
+date: '2024-06-14'
+tags: ['trip']
+draft: false 
+summary: Long week trip to Austrian alps
+images: []
+authors: ['mrded']
+---
+## When?  
+
+**Duration:** 14-23 June
+**Departure:** Friday, 14th June after work (~5 PM)
+
+## Where?
+
+**Main Destination:** Austrian Alps
+
+This is an approximate route we're planning to do
+
+![](https://habrastorage.org/webt/mq/ef/so/mqefso6-edsqwsyn9osntllusky.jpeg)
```

Comment:
Although I would suggest the direct route from Luxembourg via Stuttgart and onto Austria